test/code/providers/Container_ImageInventory_Class_Provider_UnitTest.cpp

```diff
@@ -155,14 +155,15 @@ class ImageInventoryTest : public CppUnit::TestFixture
 		// Should have same number of images
 		CPPUNIT_ASSERT_EQUAL(images.size(), context.Size());
 
-		wchar_t currentId[66];
+		wchar_t currentId[128];
 		int imageCount = 0;
 
 		// Verify every field of every object
 		for (unsigned i = 0; i < images.size(); i++)
 		{
 			bool flag = false;
-			mbstowcs(currentId, cJSON_GetObjectItem(images[i], "InstanceID")->valuestring, 65);
+			int rc = mbstowcs(currentId, cJSON_GetObjectItem(images[i], "InstanceID")->valuestring, (sizeof(currentId)/sizeof(currentId[0])) - 1;
```

Comment:
I think you have unbalanced parenthesis here. Please verify that, with your fix, it still compiles and it fixes the problem (i.e. the unit test that this affects is fixed).